### PR TITLE
Local Contexts on Collection

### PR DESCRIPTION
**Pull request by lamtu1**
`FEATURE:` Add local contexts field into the metadata of the collection page.

`TO-DO:`
- [x] Add in the `metadata` into the `collection form`
- [x] Check to see if the `LC` exist in `@collection`
- [x] Hook it up to the display to present the LC

### CODE REVIEW

**Comment by lamtu1** [comment]
fixes #3029

**Review by CGillen — CHANGES_REQUESTED** [review]
I'm so sorry. Could you make `local_contexts` singular to follow the rest of the metadata properties? => `local_context`

**Comment by jsimic** [comment]
@CGillen Local Contexts is an organization, plural: [https://localcontexts.org/](https://localcontexts.org/)

File: app/views/hyrax/collections/show.html.erb [hunk 1]
```diff
@@ -114,6 +114,14 @@
     <button type="submit" form="collection-search" class='btn btn-lg btn-primary collection-landing-button col-sm-10'><i class='glyphicon glyphicon-search'></i> Search</button>
   <% end %>
 
+  <!-- Create render display for the Local Contexts Icon -->
+  <% if !@collection.local_contexts.blank? || !@collection.local_contexts.empty? %>
```
Comment (CGillen) [inline app/views/hyrax/collections/show.html.erb:118]: `#blank?` should act the same as `#empty?` across empty arrays, empty strings and doesn't trip up on `nil`. You shouldn't need the call to `#empty?` unless I missed something

File: app/views/oregon_digital/base/_local_contexts.html.erb [hunk 2]
```diff
@@ -8,7 +8,9 @@
     <%# FETCH: Get the array of the local contexts from work/collection %>
     <% if location_name.eql?('collection') %>
       <h2 class='h6'>Local Contexts</h2>
-      <%# DISPLAY COLLECTION CODE WILL GO HERE ON ANOTHER PR %>
+      <% @local_contexts_labels = OregonDigital::LocalContextsService.new.all_labels(collection_lc.local_contexts) %>
+      <% @lc_labels = @local_contexts_labels %>
```
Comment (CGillen) [inline app/views/oregon_digital/base/_local_contexts.html.erb:12]: Extra assignment?

**Review by CGillen — CHANGES_REQUESTED** [review]

File: app/views/oregon_digital/base/_local_contexts.html.erb (hunk 2 above)
Comment (lamtu1) [reply app/views/oregon_digital/base/_local_contexts.html.erb:12]: I have to pass in an extra `assignment` as the row display at the bottom of the code utilize the `@lc_labels`. So to make sure that one section on display work for both `work & collection`, I have to do that extra assignment to make the display universal usage in both area of `work & collection`.

File: app/views/hyrax/collections/show.html.erb (hunk 1 above)
Comment (lamtu1) [reply app/views/hyrax/collections/show.html.erb:118]: Got it. I will go in and see through all the code and remove those extra `empty?` code, glad to know `blank?` does all of that, I thought it only handle `nil`

**Review by CGillen — APPROVED** [review]

**Pull request merged**